db/migrate/20230211171544_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+
+      t.timestamps
+    end
```

Comment:
- [ ] You are missing a number of columns here required by dev. When I run migrations and run the rails server, I get an error message as seen in the image below because the email field does not exist in the schema. Your schema has all these required fields but the migrations do not. This also causes further error messages as a reviewer is going through your project.
![image](https://user-images.githubusercontent.com/45460192/218275837-aa1822b7-fd89-4342-987c-74b55bac19f1.png)

Reply:
Fixed.

Reply:
Thanks for the feedback @Lucas-Masaba